linkedsensor/ontology/.htaccess

```diff
@@ -0,0 +1,27 @@
+Options +FollowSymLinks
+# Turn off MultiViews
+Options -MultiViews
+
+# Directive to ensure *.rdf files served as appropriate content type,
+# if not present in main apache config
+AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+
+RewriteEngine on
+
+#Rewrite rules for Asteroids vocabulary extension 
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml|text/\*|\*/\*)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_ACCEPT} text/\* [OR]
+RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+
+#RewriteRule ^asteroids$ http://dgarijo.github.io/VideoGameOntology/GameExtensions/AsteroidsOntology/asteroidsDoc/  [R=303,NE,L]
+#RewriteCond %{HTTP_ACCEPT} ^.*application/rdf\+xml.* 
+#RewriteRule ^asteroids$ http://dgarijo.github.io/VideoGameOntology/GameExtensions/AsteroidsOntology/asteroids.owl [R=303,NE,L]
+#RewriteCond %{HTTP_ACCEPT} ^.*text/turtle.* 
+#RewriteRule ^asteroids$ http://dgarijo.github.io/VideoGameOntology/GameExtensions/AsteroidsOntology/asteroids.ttl [R=303,NE,L]
+
+#default response: owl
+RewriteRule ^asteroids$ http://linkedsensor.in/ontology/agrion.owl [R=303,NE,L]
```

Comment:
Similar copy & paste issue.

Reply:
I'm guessing "asteroids" came from "games/.htaccess"
I don't know what mapping you want here.  Those source patterns are relative to the current directory, so in this case:
https://w3id.org/linkedsensor/ontology/asteroids => http://linkedsensor.in/ontology/agrion.owl
Maybe you want
```
RewriteRule ^$ http://linkedsensor.in/ontology/agrion.owl [R=303,NE,L]
```
or
```
RewriteRule ^agrion$ http://linkedsensor.in/ontology/agrion.owl [R=303,NE,L]
```
Did you want to use https on your site?

Reply:
ok, I got it. thank you for your help.